packages/css-data/src/property-parsers/filter.ts

```diff
@@ -0,0 +1,63 @@
+import * as csstree from "css-tree";
```

Comment:
Where is the test?

Reply:
Can't test it, as i added a comment in filter parser. The css-tree parser can't validate `filter`. So, i am relying on a validator which uses `CSSStyleSheet` which is available only in browser. Should we polyfill with `jsdom` and then add tests. Currently it's not available, so it keeps failing 👍

Reply:
Probably add throw in non browser env then?

Reply:
Then i am not sure, if it's helping in writing tests. Basically, we are not polyfiling any browser env here. So, the tests always throws error. And we are using inbuilt browser `CSSStyleValue` function itself and there is not custom logic that tests actually run in local.